### ISSUE TITLE
'Invalid Dangerfile' after Fastlane 2.172.0 update

### ISSUE THREAD

**Issue by imaginaris** [nakiostudio/danger-xcov#35]
In one of our projects after upgrading to fastlane 2.172.0 we are getting an error:
```
[!] Invalid Dangerfile file: Could not find option skip_package_dependencies_resolution in the list of available options: 
workspace, project, scheme, configuration, source_directory, derived_data_path, xccov_file_direct_path, output_directory, 
cloned_source_packages_path, html_report, markdown_report, json_report, minimum_coverage_percentage, slack_url, 
slack_channel, skip_slack, slack_username, slack_message, ignore_file_path, include_test_targets, exclude_targets, 
include_targets, only_project_targets, disable_coveralls, coveralls_service_name, coveralls_service_job_id, 
coveralls_repo_token, xcconfig, ideFoundationPath, legacy_support
```

Our Dangerfile does not specify `skip_package_dependencies_resolution` key.
After downgrading fastlane to 2.171.0 the issue is gone.
Is it an issue with Fastlane or danger-xcov needs to be updated to accomodate new changes?

Versions we are using:
```
danger 8.2.2
fastlane 2.172.0
xcov 1.7.5
danger-xcov 0.5.0
xcode 12.3
```

Build log:
https://app.bitrise.io/build/3f11bbf9ba9c46a7#?tab=log

Dangerfile:
https://github.com/rakutentech/ios-inappmessaging/blob/master/Dangerfile

**Comment by escakot** [comment]
Same here. Also encountering this.

**Comment by k0nserv** [comment]
Same, I was trying to look through the relevant fastlane code, but I am not sure I understand it. From reading it it seems that the issue happens when `skip_package_dependencies_resolution` is passed to `report`, but in my code I didn't pass it at all 🤔

**Comment by imaginaris** [comment]
It seems like the issue has been fixed in `2.174.0` so only `2.172.0` and `2.173.0` have this problem.
